<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="3" tests="2" time="919.628">
 <testsuite name="Alerts: Creating Frozen Data models" timestamp="2019-12-02T19:47:28" hostname="localhost" time="919.628" errors="0" tests="2" skipped="0" disabled="0" failures="3">
  <testcase classname="Alerts: Creating Frozen Data models" name="Creating Frozen Data Model" time="860.248">
   <failure type="toBe" message="Expected false to be true, &apos;Model Build was not successful&apos;."><![CDATA[Error: Failed expectation
    at Object.<anonymous> (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\apps\e2e\src\specs\Alerts\spec04_Creating_Frozen_Data_Model.ts:161:51)
    at step (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:133:27)
    at Object.next (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:114:57)
    at fulfilled (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:104:62)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)]]>
   </failure>
  </testcase>
  <testcase classname="Alerts: Creating Frozen Data models" name="Deleting Frozen Data model" time="28.494">
   <failure type="toBe" message="Expected false to be true, &apos;Deleted models was not successfull&apos;."><![CDATA[Error: Failed expectation
    at Object.<anonymous> (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\apps\e2e\src\specs\Alerts\spec04_Creating_Frozen_Data_Model.ts:177:81)
    at step (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:133:27)
    at Object.next (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:114:57)
    at fulfilled (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:104:62)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)]]>
   </failure>
   <failure type="toBe" message="Expected false to be true, &apos;Deleted models was not successfull&apos;."><![CDATA[Error: Failed expectation
    at Object.<anonymous> (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\apps\e2e\src\specs\Alerts\spec04_Creating_Frozen_Data_Model.ts:183:8)
    at step (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:133:27)
    at Object.next (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:114:57)
    at fulfilled (d:\AtonixAzureRepo\New_Repo_Atonix_Automation\atx-test-automation\node_modules\tslib\tslib.js:104:62)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>